Install uyuni and adoc ke (#880)
SUSE/doc-susemanager: modules/installation/pages/install-uyuni.adoc

[[install-uyuni]]
= Install Uyuni Server with openSUSE

{productname} Server can be installed on openSUSE.

> [NOTE]

Reviewer: I would take this out of an admonition, and make it part of the stem sentence to the actual table.

Author: Ok, I'll submit such a patch separately.

====
This table specifies the minimum requirements.
====

> In the following table replace _version_ with the actual product version string.

Reviewer: ```suggestion
In this table, replace _version_ with the version of the product you are using.
```

Reviewer: But I don't see "version" in the table?

[cols="1,1", options="header"]
.Hardware and Software Requirements
|===
| Requirement            | Description
| openSUSE Leap 15.1     | Clean installation, up-to-date
| Multi-core CPU         | x86_64
| 8 GB RAM *) 	         | Sufficient for testing purposes
| 100 GB Disk Space **)  | Disk space depends on your channel requirements
|===

*) 32 GB of RAM recommended for a production environment (16 GB for a base installation).

> **) 50 GB per SUSE or openSUSE product and 360 GB per Red Hat product.

Reviewer: I'm not sure what the close parentheses are doing here. You might also like to consider replacing the asterisks with proper icons. We use `icon:asterisk[role="none"]` elsewhere in the suite.

Author: Ok, I will go for `icon:asterisk[role="none"]`.  (Without something directly after the *, adoc treats it as list item. Maybe, I coud have escaped them with backslashes...)

////
[NOTE]
====
Storage size values are the absolute minimum—only suitable for a small test or demo installation.
Especially [path]``/var/spacewalk/`` may quickly need more space.
Also consider to create a separate partition for [path]``/srv`` where Kiwi images are stored.
====
////

////
[cols="1,1,1", options="header"]
|===
| VirtIO Storage Disks | Name      | Sizing
| VirtIO Disk 2        | spacewalk | 101{nbsp}GB
| VirtIO Disk 3        | pgsql     | 50{nbsp}GB
| VirtIO Disk 4        | swap      | 4{nbsp}GB
|===
////

> . Install openSUSE Leap 15.1.

Reviewer: ```suggestion
== Install openSUSE Leap 15.1.
```

If you want to re-use an existing openSUSE Leap 15.1 installation make sure that all available package updates are installed.

. Configure a resolvable fully quallified domain name (FQDN) with menu:yast[System > Netwok Settings > Hostname/DNS].

. Add the repository for installing the {productname} Server software as [systemitem]``root``:
+

// varaible assignment to avoid overloang lines
+
----
repo=repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-4.0-POOL-x86_64-Media1/
zypper ar https://download.opensuse.org/$repo uyuni-server-stable
----

. Refresh metadata from the repositories as [systemitem]``root``:
+

----
zypper ref
----

. Install the pattern for the {productname} Server as [systemitem]``root``:
+

----
zypper in patterns-uyuni_server
----

. Reboot.

> For proceeding with {productname} setup, see xref:installation:uyuni-server-setup.adoc[Uyuni Setup].

Reviewer: ```suggestion
When this installation is complete, you can continue with {productname} setup.
For more information, see xref:installation:uyuni-server-setup.adoc[].
```

[[install-uyuni]]
= Install Uyuni Server with openSUSE

{productname} Server can be installed on openSUSE.

[NOTE]
====
This table specifies the minimum requirements.
====

> In this table, replace ``version`` with the version of the product you are using.

Reviewer 2: What does this mean? I don't see ```version``` at the table.

Author: I commented it.

[cols="1,1", options="header"]
.Hardware and Software Requirements
|===
| Requirement            | Description
| openSUSE Leap 15.1     | Clean installation, up-to-date
| Multi-core CPU         | x86_64
| 8 GB RAM *) 	         | Sufficient for testing purposes
| 100 GB Disk Space **)  | Disk space depends on your channel requirements
|===

*) 32 GB of RAM recommended for a production environment (16 GB for a base installation).

**) 50 GB per SUSE or openSUSE product and 360 GB per Red Hat product.

> ////

Reviewer 2: Should we remove this?

Author: @juliogonzalez What do you want to remove?  The storage space requirements or the slashes (`////`)?  The slashes are a block comment start marker.  It makes the following NOTE invisible.  I've no strong opinion on those.

[NOTE]
====
Storage size values are the absolute minimum—only suitable for a small test or demo installation.
Especially [path]``/var/spacewalk/`` may quickly need more space.
Also consider to create a separate partition for [path]``/srv`` where Kiwi images are stored.
====
////

////
[cols="1,1,1", options="header"]
|===
| VirtIO Storage Disks | Name      | Sizing
| VirtIO Disk 2        | spacewalk | 101{nbsp}GB
| VirtIO Disk 3        | pgsql     | 50{nbsp}GB
| VirtIO Disk 4        | swap      | 4{nbsp}GB
|===
////

== Install openSUSE Leap 15.1.

> If you want to re-use an existing openSUSE Leap 15.1 installation make sure that all available package updates are installed.

Reviewer 2: The instance has to be brand new and fully updated. You just said it above :-)

Author: ok, I changed it for clarity.

. Configure a resolvable fully quallified domain name (FQDN) with menu:yast[System > Netwok Settings > Hostname/DNS].

. Add the repository for installing the {productname} Server software as [systemitem]``root``:
+

// varaible assignment to avoid overloang lines
+
----

> repo=repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-4.0-POOL-x86_64-Media1/

Reviewer 2: Should we mention how to add Master, or at least point to https://www.uyuni-project.org/pages/devel-version.html ?

Author: The install and setup procedure is already detailed and complicate enough.  If really needed, we should add a dedicated section about Master the install and setup chapter.  I think this would require a separate issue.

Reviewer 2: I don't really think we need such fancy things. A simple note pointing to the URL and mentioning that using devel is not supported, should be enough.

Author: ok, I'll add such pointer for stable and devel.

Reviewer 2: I can't see it added.
